psycop/common/model_training/config_schemas/data.py

```diff
@@ -27,7 +27,7 @@ class ColumnNamesSchema(PSYCOPBaseModel):
 class DataSchema(PSYCOPBaseModel):
     """Data configuration."""
 
-    dir: Union[Path, str]  # Location of the dataset # noqa
+    dir: list[Path | str] | Path | str  # Location of the dataset # noqa
```

Comment:
```suggestion
    dir: list[DirectoryPath] | Path  # Location of the dataset # noqa
```

https://docs.pydantic.dev/1.10/usage/types/#pydantic-types